doc/source/user_guide/ug_implementation.rst

```diff
@@ -759,7 +821,8 @@ is now a character string corresponding to ``histfreq`` or ‘x’ for none.
 files, no matter what the frequency is.) If there are no namelist flags
 with a given ``histfreq`` value, or if an element of ``histfreq_n`` is 0, then
 no file will be written at that frequency. The output period can be
-discerned from the filenames.
+discerned from the filenames.  Because all history is average output, it's
+not possible to write instananeous output at any frequency except every timestep.
```

Comment:
I don't have the answer to this question, but it's a good one.  I'm not super familiar with all the details of the history mechanism.  @dabail10.

Reply:
This is a good question. When I implemented the multiple stream capability years ago, I did not think about the hist_avg variable. I kind of just made the assumption that it was all or none for the averaging. So, if hist_avg = .false. all streams should be instantaneous. However, I think it would be nice to make this an array, so that we could pick and choose for each stream. There is also some issues in the code where it checks only the first stream for a value that is not 'x'. This means that the first one must always be defined or it turns off history completely.

Reply:
OK, so if I understand correctly, the doc as modified by this PR is wrong ?

Reply:
I haven't actually tested the code with hist_avg = .false.